### PR TITLE
adding new python script to process the real excel files

### PR DESCRIPTION
**Pull request by GhaithAshkar**
- adding new python script called "Server.py"
- changing the old python file name from "server.py" to "testing_server.py"

### CODE REVIEW

File: testing_server.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# import sys
```
Comment (Silver2er0) [inline testing_server.py:1]: Is this file still needed? If not, delete it.

File: server.py [hunk 2]
```diff
@@ -1,105 +0,0 @@
-import sys
```
Comment (Silver2er0) [inline server.py:1]: same here

File: .idea/vcs.xml
Comment (Silver2er0) [inline .idea/vcs.xml:1]: Don't upload unnecessary files like these.

Comment (GhaithAshkar) [reply .idea/vcs.xml:1]: ok

File: testing_server.py (hunk 1 above)
Comment (GhaithAshkar) [reply testing_server.py:1]: you can delete it ...

File: server.py (hunk 2 above)
Comment (GhaithAshkar) [reply server.py:1]: i actually  deleted it, it's in the red color

File: .idea/vcs.xml
Comment (GhaithAshkar) [reply .idea/vcs.xml:1]: i added all those files to  gitignore file
